backend/src/main/kotlin/org/icpclive/data/DataBus.kt

```diff
@@ -16,11 +17,16 @@ object DataBus {
     val contestInfoFlow = CompletableDeferred<Flow<ContestInfo>>()
     val mainScreenFlow = CompletableDeferred<Flow<MainScreenEvent>>()
     val queueFlow = CompletableDeferred<Flow<QueueEvent>>()
+    val queueFeaturedRunsFlow = MutableSharedFlow<Int>( // flow of run ids that need to be braking news
```

Comment:
Can it be registered by service filling it? Or you think it should be filled by several services?

Reply:
I think Analytics Service is optional and we will disable it sometimes. Therefore, Queue may be no depend on analytics

Reply:
Okey, let's keep it for now, I'll think about it later.